### PR TITLE
special-interest-groups: add "Repositories, Projects and labels"

### PR DESCRIPTION
**Pull request by zimulala**
Add "Repositories, Projects and labels"

### CODE REVIEW

**Comment by zimulala** [comment]
PTAL @tangenta @AilinKid

File: special-interest-groups/sig-ddl/README.md [hunk 1]
```diff
@@ -34,6 +34,17 @@ See [SIG DDL Member List](./member-list.md)
 
 * Slack: channel #sig-ddl in the [tidbcommunity](https://pingcap.com/tidbslack) slack workspace, you can join this channel through [this invitation link](https://slack.tidb.io/invite?team=tidb-community&channel=sig-ddl&ref=pingcap-community).
 
+## Repositories, Projects and labels
```
Comment (AilinKid) [inline special-interest-groups/sig-ddl/README.md:37]:
```suggestion
## Repositories, Projects and Labels
```

**Comment by zimulala** [comment]
PTAL @tangenta @AilinKid

**Comment by zimulala** [comment]
PTAL @tangenta @AilinKid

**Pull request merged**